core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -530,7 +530,6 @@ object StorageTool extends Logging {
     metaPropertiesEnsemble.verify(metaProperties.clusterId(), metaProperties.nodeId(),
       util.EnumSet.noneOf(classOf[VerificationFlag]))
 
-    stream.println(s"metaPropertiesEnsemble=$metaPropertiesEnsemble")
```

Comment:
`testFormatEmptyDirectory` and `testFormatSucceedsIfAllDirectoriesAreAvailable` need to be updated for this change.

Reply:
@cmccabe not sure whether this output is debugging-only? it was added by #14628, and the code was `System.out.println(s"metaPropertiesEnsemble=${metaPropertiesEnsemble}")`. Hence, I assume it was used to debug only before.

Reply:
@cmccabe not sure whether this output is debugging-only? it was added by #14628, and the code was `System.out.println(s"metaPropertiesEnsemble=${metaPropertiesEnsemble}")`. Hence, I assume it was used to debug only before.

Reply:
Done